common/src/test/java/io/seata/common/util/ReflectionUtilTest.java

```diff
@@ -0,0 +1,75 @@
+package io.seata.common.util;
+
+import java.io.Serializable;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ReflectionUtilTest {
+
+    @Test
+    public void testGetClassByName() throws Exception {
```

Comment:
The `throws` statement could be more specific ie. `ClassNotFoundException`

Reply:
Done